tests/acceptance/features/users.feature

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+Feature: Get the list of users from the IdM.
+
+    Scenario:  01: Get the list of trial users
+      Given a user with name "trial1", password "trial" and role "trial"
+      Given a user with name "trial2", password "trial" and role "trial"
+      Given a user with name "basic1", password "trial" and role "basic"
+      When I request a list of trial users from the Keystone
+      Then the Keystone returns a list with "2" trial users
+
+    Scenario: 02: Get the list of community users
+      Given a user with name "community1", password "password" and role "community"
+      Given a user with name "community2", password "password" and role "community"
+      Given a user with name "basic1", password "password" and role "basic"
+      When I request a list of community users from the Keystone
+      Then the Keystone returns a list with "2" community users
+
+    Scenario Outline: 03: Get list of several users
+      Given a user with name "<username>", password "<password>" and role "<role>"
```

Comment:
Not sure, if you want here a outline scenario or a scenario with a tabular data. You now that you are creating one scenario per each of the example data, don't?